primitives/src/balances.rs

```diff
@@ -195,6 +204,15 @@ impl Convert<u128, BalanceType> for EncointerBalanceConverter {
 	}
 }
 
+#[allow(non_snake_case)]
+pub fn I64F64_to_U64F64(source: I64F64) -> Option<U64F64> {
+	if source.is_negative() {
+		return None
+	}
+
+	Some(U64F64::from_le_bytes(source.to_le_bytes()))
```

Comment:
I don't like this because byte order is not what makes this work and it confused me. what you do could be done with scale encode and decode and it would be at least obvious that it is a hacky typecast

Reply:
I agree that the LE is confusing, I have made it more trivial and just use `to_bits` and `from_bits` (much better) and also made a remark about the conversion/

I don't think it is good to to use the scale codec because it is much more inefficient than using the `#[inline]` bit conversions that operate on a stack instead of on the heap.

With the `from_bits` and `to_bits` conversions, I don't even consider this hacky anymore.